packages/shared/friends/sagas.ts

```diff
@@ -480,7 +481,9 @@ function* refreshFriends() {
         socialInfo,
         friends: friendIds,
         fromFriendRequests: requestedFromIds,
-        toFriendRequests: requestedToIds
+        toFriendRequests: requestedToIds,
+        // initialize an empty map becuase there is no way to get the current statuses, the matrix client store is empty at this point
```

Comment:
Typo:
```suggestion
        // initialize an empty map because there is no way to get the current statuses, the matrix client store is empty at this point
```